src/main/kotlin/com/hibob/project/service/feedbackService.kt

```diff
@@ -0,0 +1,25 @@
+package com.hibob.project.service
+
+import com.hibob.project.dao.*
+import jakarta.ws.rs.NotFoundException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class FeedbackService @Autowired constructor(private val feedbackDao: FeedbackDao) {
+    fun getAllFeedbacks(companyId: Long): List<FeedbackConfiguration> {
+        val feedbacks = feedbackDao.getAllFeedbacks(companyId)
+
+        if (feedbacks.isEmpty())
+            throw NotFoundException("There are no feedbacks for this company")
```

Comment:
should this be an exception if my company doesn't have any feedback's yet?
Think about the user's exprience in this case